FRAuth/FRAuth/Authentication/Node.swift

```diff
@@ -322,7 +322,11 @@ public class Node: NSObject {
         var callbacks: [Any] = []
         
         for callback:Callback in self.callbacks {
-            callbacks.append(callback.buildResponse())
+            if let textOutPutCallback = callback as? TextOutputCallback, (textOutPutCallback.messageType == .unknown) {
```

Comment:
tiny comment (can be ignored): in `textOutPutCallback` P should be lowercase and the (..) can be omitted around the check for .unknown type.

Reply:
yeap it annoys me too...will fix :)

Reply:
I like the () more, instead of having it empty, in my head it defines the check better